tests/test_trade.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from poandy.controller.trade import TradeController
+from poandy.controller.account import AccountController
+
+
+class TestTradeController:
+    def test_get_trades_default(self):
+        account_id = AccountController.get_default_account_id()
```

Comment:
I think you can refactor the code by putting account_id as a variable within the scope of the class instead of within the scope of each function.

Reply:
thanks for the suggestion. Will do!